src/main/java/it/gov/pagopa/admissibility/dto/ErrorDTO.java

```diff
@@ -0,0 +1,18 @@
+package it.gov.pagopa.admissibility.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import it.gov.pagopa.admissibility.exception.Severity;
+import lombok.Value;
+
+import javax.validation.constraints.NotBlank;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Value
+public class ErrorDTO {
```

Comment:
copy it again from transaction, this version will give you some trouble during tests